docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,494 @@
+par_kB = PEtabParameter(:kB)
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+params = [par_kB, par_kD, par_kP]
+
+c1 = Dict(:kB => 1.0)
+c2 = Dict(:kB => 0.5)
+simulation_conditions = Dict("c1" => c1, "c2" => c2)
+
+using DataFrames
+m1 = DataFrame(simulation_id="c1", obs_id="obs_P", time=t1, measurement=d1)
+m2 = DataFrame(simulation_id="c2", obs_id="obs_P", time=t2, measurement=d2)
+measurements = vcat(m1,m2)
+
+petab_model = PEtabModel(rn, simulation_conditions, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+Note that the `u0` we pass into `PEtabModel` through the `state_map` argument no longer contains the value of *S* (as it is provided by the conditions). 
+
+### Varying parameters between different simulation conditions
+Sometimes, the parameters that are used vary between the different conditions. Consider our catalysis example, if we had performed the experiment twice, using two different enzymes with different catalytic properties, this could have generated such conditions. The two enzymes could e.g. yield different rates (*kP₁* and *kP₂*) for the `SE --> P + E` reaction, but otherwise be identical. Here, the parameters *kP₁* and *kP₂* are unique to their respective conditions. PEtab.jl provides support for cases such as this, and [its documentation](https://sebapersson.github.io/PEtab.jl/stable/Julia_condition_specific/) provided instructions of how to handle them.
+
+## [Additional features: Initial conditions](@id petab_simulation_initial_conditions)
+
+### [Fitting initial conditions](@id petab_simulation_initial_conditions_fitted)
+Sometimes, initial conditions are uncertain quantities which we wish to fit to the data. This is possible by [defining an initial condition as a parameter](@ref dsl_description_parametric_initial_conditions): 
+```@example petab4
+using Catalyst, PEtab # hide
+rn = @reaction_network begin
+    @parameters E0
+    @species E(t)=E0
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
+end
+nothing # hide
+```
+Here, the initial value of `E` is equal to the parameter `E0`. We modify our `u0` vector by removing `E` (which is no longer known):
+```@example petab4
+u0 = [:S => 1.0, :SE => 0.0, :P => 0.0]
+nothing # hide
+```
+Next, we add `E0` to the parameters we wish to fit:
+```@example petab4
+par_kB = PEtabParameter(:kB)
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+par_E0 = PEtabParameter(:E0)
+params = [par_kB, par_kD, par_kP, par_E0]
+nothing # hide
+```
+and we can use our updated `rn`, `u0`, and `params` as input to our `PEtabModel`.
+
+### [Uncertain initial conditions](@id petab_simulation_initial_conditions_uncertainty)
+Often, while an initial condition has been reported for an experiment, its exact value is uncertain. This can be modelled by making the initial condition a [parameter that is fitted to the data](@ref petab_simulation_initial_conditions_fitted) and attach a prior to it corresponding to our certainty about its value. 
```

Comment:
```suggestion
Often, while an initial condition has been reported for an experiment, its exact value is uncertain. This can be modelled by making the initial condition a [parameter that is fitted to the data](@ref petab_simulation_initial_conditions_fitted) and attaching a prior to it corresponding to our certainty about its value. 
```